conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -486,7 +488,7 @@ public void testBinaryResponseClientDisconnect() throws Exception {
                     .close();
             // Unfortunately there's no great way to tell when the server has finished processing
             // our request.
-            Uninterruptibles.sleepUninterruptibly(Duration.ofMillis(500));
+            Uninterruptibles.sleepUninterruptibly(Duration.ofMillis(100));
```

Comment:
I think this test was implemented before CojureHandler allowed custom ExceptionHandler implementations to be provided in the builder. Let's move this single test into a new class which sets up the undertow server using a delegating wrapper around the exception handler, that way we can use a countdownLatch after the exception handler has been called instead of an arbitrary sleep or retry.